### PR TITLE
docs: Differentiate between workspace and project catalog upgrades

### PR DESCRIPTION
**Pull request by cbuto**
## Description of changes being made
This PR splits up the catalog upgrade docs between project and workspaces. As a user, if I have a workspace application, I would expect the upgrade docs to live within the workspace documentation. The project documentation should also only reference how to upgrade project catalog applications imo.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4281.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md [hunk 1]
```diff
@@ -29,24 +33,8 @@ Follow these steps to upgrade an application from the DKP UI:
...
-
-    ```bash
-    dkp upgrade appdeployment <app-name> --workspace=my-workspace --project=my-project --to-version=<version.number>
-    ```
-
-For example, the following command upgrades the Kafka Operator application in a workspace to version `0.20.2`:
-
 ```bash
-dkp upgrade appdeployment kafka-operator --workspace=my-workspace --to-version=0.20.2
+dkp upgrade appdeployment <app-name> --workspace=my-workspace --project=my-project --to-version=<version.number>
```
Comment (gracedo) [inline pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md:37]:
```suggestion
dkp upgrade catalogapp <appdeployment-name> --workspace=my-workspace --project=my-project --to-version=<version.number>
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/upgrading-applications/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+1.  Select the three dot button from the bottom-right corner of the desired application tile, and then select **Edit**.
+
+1.  Select the **Version** drop-down, and select a new version. This drop-down will only be available if there is a newer version to upgrade to.
+
+1.  Select **Save**.
+
+### Upgrade with CLI
+
+```bash
+dkp upgrade appdeployment <app-name> --workspace=my-workspace --to-version=<version.number>
```
Comment (gracedo) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/upgrading-applications/index.md:33]:
```suggestion
dkp upgrade catalogapp <appdeployment-name> --workspace=my-workspace --to-version=<version.number>
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/upgrading-applications/index.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+```
+
+For example, the following command upgrades the Kafka Operator application in a workspace to version `0.20.2`:
+
+```bash
+dkp upgrade appdeployment kafka-operator --workspace=my-workspace --to-version=0.20.2
```
Comment (gracedo) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/upgrading-applications/index.md:39]:
```suggestion
dkp upgrade catalogapp kafka-operator-abc --workspace=my-workspace --to-version=0.20.2
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/upgrading-applications/index.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+1.  Select **Save**.
+
+### Upgrade with CLI
+
+```bash
+dkp upgrade appdeployment <app-name> --workspace=my-workspace --to-version=<version.number>
+```
+
+For example, the following command upgrades the Kafka Operator application in a workspace to version `0.20.2`:
```
Comment (gracedo) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/upgrading-applications/index.md:36]:
```suggestion
For example, the following command upgrades the Kafka Operator application, named `kafka-operator-abc`, in a workspace to version `0.20.2`:
```

**Review by gracedo — COMMENTED** [review]
we've updated the cli command to be `upgrade catalogapp` instead of `upgrade appdeployment`

File: pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md (hunk 1 above)
Comment (cbuto) [reply pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md]: good catch! fcfc2217b513430dfb494ca32da5e2c43a2f6293

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/upgrading-applications/index.md (hunk 2 above)
Comment (cbuto) [reply pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/upgrading-applications/index.md]: fcfc2217b513430dfb494ca32da5e2c43a2f6293

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/upgrading-applications/index.md (hunk 4 above)
Comment (cbuto) [reply pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/upgrading-applications/index.md]: fcfc2217b513430dfb494ca32da5e2c43a2f6293

**Review by mwrightd2 — APPROVED** [review]
LGTM

**Comment by cmastr** [comment]
@cbuto these docs also live in the Konvoy docs now so we need to copy it there.

**Review by cmastr — APPROVED** [review]

**Review by takirala — APPROVED** [review]

**Comment by cbuto** [comment]
@cmastr I updated the overview links in both kommander and konvoy docs. I think copying these pages to konvoy needs some more discussion as to where they would go, etc. Right now, I don't think there's a good spot for them. It might even result in restructuring the existing upgrade docs.

File: pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md [hunk 5]
```diff
@@ -19,7 +19,11 @@ Follow these steps to upgrade an application from the DKP UI:
 
 1.  From the top menu bar, select your target workspace.
 
-1.  Select **Applications** from the sidebar menu.
+1.  From the side menu bar, select **Projects**.
```
Comment (cwyl02) [inline pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md:22]: should we put the content around project catalog app upgrade to `/projects/applications/catalog-applications/upgrading-applications/index.md` just like the workspace catalog app one's?

Comment (cbuto) [reply pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md:22]: yeah I can move it! one sec

Comment (cbuto) [reply pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md]: done in 60f8e8d28d40c86f8aa7ed89503b1513013873ce

**Review by cwyl02 — APPROVED** [review]

**Review by nataliepina — APPROVED** [review]

**Pull request merged**